test/extensions/tracers/opencensus/tracer_test.cc

```diff
@@ -166,12 +166,12 @@ TEST(OpenCensusTracerTest, PropagateTraceContext) {
   // The test calls the helper with each kind of incoming context in turn.
   auto helper = [](const std::string& header, const std::string& value) {
     OpenCensusConfig oc_config;
-    oc_config.add_incoming_trace_context(OpenCensusConfig::trace_context);
-    oc_config.add_incoming_trace_context(OpenCensusConfig::grpc_trace_bin);
-    oc_config.add_incoming_trace_context(OpenCensusConfig::cloud_trace_context);
-    oc_config.add_outgoing_trace_context(OpenCensusConfig::trace_context);
-    oc_config.add_outgoing_trace_context(OpenCensusConfig::grpc_trace_bin);
-    oc_config.add_outgoing_trace_context(OpenCensusConfig::cloud_trace_context);
+    oc_config.add_incoming_trace_context(OpenCensusConfig::TRACE_CONTEXT);
```

Comment:
Optional: add NONE as both incoming and outgoing to smoke-test that it's ignored.

Reply:
Done